Cargo.toml

```diff
@@ -22,6 +22,6 @@ path = "src/main.rs"
 travis-ci = { repository = "loony-bean/textplots-rs", branch = "master" }
 
 [dependencies]
-drawille = "0.2.3"
+drawille = "0.3.0"
```

Comment:
Thank you @BroderickCarlin for adding this functionality, it looks really great!

If it's not too much, can I ask you to update few things in the code? Will put comments inline